### PR TITLE
MINOR: fix example usage of WindowedSerdes

### PR DESCRIPTION
**Pull request by vvcephei**

### CODE REVIEW

File: src/main/java/io/confluent/examples/streams/TopArticlesLambdaExample.java [hunk 1]
```diff
@@ -50,7 +50,7 @@
  *
  * In this example, we count the TopN articles from a stream of page views (aka clickstreams) that
  * reads from a topic named "PageViews". We filter the PageViews stream so that we only consider
- * pages of type article, and then map the recored key to effectively nullify the user, such that
+ * pages of type article, and then map the recorded key to effectively nullify the user, such that
```
Comment (mjsax) [inline src/main/java/io/confluent/examples/streams/TopArticlesLambdaExample.java]: I think "record key" was correct -- the key of the record.

Comment (vvcephei) [reply src/main/java/io/confluent/examples/streams/TopArticlesLambdaExample.java]: oh, shoot. I'll fix it.

**Comment by vvcephei** [comment]
@mjsax I fixed that typo.

Also, I increased the timeout of TopArticlesLambdaExampleTest to 60s. We had one build that filed because it hit the old timeout of 30s. The subsequent build passed after 25s, so I think 30s is just too close.

**Comment by mjsax** [comment]
Thanks @vvcephei ! Sounds good!

**Pull request merged**